app/controllers/registrations_controller.rb

```diff
@@ -22,7 +22,8 @@ def account_update_params
       :agency_id,
       :password,
       :password_confirmation,
-      :current_password
+      :current_password,
+      :approved
```

Comment:
@joedean I reverted that commit -- we'll keep going with turbolinks for now.  Note that we're using the jquery-turbolinks gem to make it so you shouldn't need to do anything special for turbolinks to work.